src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -464,14 +464,16 @@ public class JabRefPreferences implements PreferencesService {
     private JournalAbbreviationPreferences journalAbbreviationPreferences;
     private FieldPreferences fieldPreferences;
 
-    // The constructor is made private to enforce this as a singleton class:
-    private JabRefPreferences() {
+    /**
+     * The constructor is made package private to enforce this as a singleton class while allowing testing of the class
+     */
+    JabRefPreferences() {
```

Comment:
I would need to rewrite "get instance". -- A new method "reset references"
The intention is that the preferences are created from scratch at each test. No reuse.

The main paint is about re initialization each time in the tests

Reply:
Reading your next comment, the whole change is unnecessary. However, I wonder why there are no tests for preferences at all.

Reply:
Right now the preferences work live on the real preferences data. there are no tests at all, because the whole class is still untestable. This is one driver for me to refactor the whole preferences mechanic. So we have a testable part that can be instantiated with a mock dao (?) and the real java preferences.